EDGAR/embedder.py

```diff
@@ -0,0 +1,125 @@
+"""
+Embedder class embeds phrases and returns vectors for machine learning processing
+"""
+import pandas as pd
+
+import gensim
+from gensim.models import Word2Vec
+from gensim.test.utils import common_texts
+
+import numpy as np
+
+import re
+import string
+import nltk
+from nltk.stem import PorterStemmer
+import heapq
+
+
+import torch
+from torch.utils.data import TensorDataset, DataLoader, RandomSampler, SequentialSampler
+from pytorch_pretrained_bert import BertTokenizer, BertConfig
+from pytorch_pretrained_bert import BertAdam, BertForSequenceClassification
+
+
+
+"""
+Class embedder takes in three diffeent types: 
+Word2Vec, BagOfWords, BERT 
+""""
+class embedder:
+    def __init__(self, type='BagOfWords', MAX_LEN=100):
+        
+        self.type = type
+        self.MAX_LEN = MAX_LEN
+        
+        if type == 'Word2Vec':
+            pretrained_embeddings_path = 'GoogleNews-vectors-negative300.bin'
+            self.model = gensim.models.KeyedVectors.load_word2vec_format(pretrained_embeddings_path,  binary=True)
+        elif type == 'BagOfWords':
+            self.model = None
+            self.dictionary = {}
+        elif type == 'BERT':
+            self.model = None
```

Comment:
You should put BERT.load_from_pretrained here, so it loads [Once] and not on [Every Sentence you input]